pages/posts/flight-of-a-byte.tsx

```diff
@@ -0,0 +1,112 @@
+import Head from "next/head"
+import Navbar from "../components/Navbar/Navbar"
+import Footer from "../components/Footer/Footer"
+import styles from "../../styles/Home.module.scss"
+import React from "react"
+import hljs from "highlight.js"
+import content from "./content.json"
+
+export default function Post() {
+  const posts = content.items.find((i) => i.id === "4")
+
+  React.useEffect(() => {
+    const highlight = async () => {
+      await hljs.highlightAll()
+    }
+    highlight()
+  }, [])
+
+  return (
+    <>
+      <Head>
+        <title>Wesh Network</title>
+        <meta name="description" content="Wesh Network" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.Body}>
+        <Navbar />
+        <section className={styles.SectionBlog}>
+          <div className={styles.SectionBlogContainer}>
+            <div className={styles.SectionBlogCard}>
+              <div className={styles.SectionBlogText}>
+                <h2>Wesh: Flight of a Byte</h2>
+                <p>
+                In the previous blog post, we made an example app where the Wesh user account of <code>“client2”</code> sends a message <code className="language-go">“Hello”</code> to a Contact group which is received by the other user account of <code>“client1”</code>. You can use the example to start building your Wesh app, but for development and debugging it helps to understand more about how it works. So this blog post will be a theory dive into some of the details of Wesh communication.
+                </p>
+                <p>
+                When the Wesh app on the computer for <code>client2</code> sends the bytes of a message, what is the <code className="language-go">“flight path”</code> of the bytes of <code className="language-go">“Hello”</code> as they travel to the computer of <code>client1</code>? It&apos;s different from “traditional” networking where, for example, a computer sends the message in a UDP packet to the IP address of the other computer, as described in this Wikipedia article. Instead of a specific IP address, Wesh communication happens between libp2p peers, where the client&apos;s PeerID is the value that we&apos;ve seen in previous examples and is the same no matter how the client is connected to the network.
```

Comment:
No wikipedia article is linked here

Reply:
@iuricmp , please add the link from the Notion draft.

Reply:
These should be a link for "this Wikipedia article", and another link for "libp2p peers".